walk/config.go

```diff
@@ -33,14 +33,27 @@ import (
 	gzflag "github.com/bazelbuild/bazel-gazelle/flag"
 )
 
+// generationModeType represents one of the generation modes.
+type generationModeType string
+
+// Generation modes
+const (
+	// Update: update and maintain existing BUILD files
+	generationModeUpdate generationModeType = "update_only"
+
+	// Create: create new and update existing BUILD files
+	generationModeCreate generationModeType = "create"
```

Comment:
I think we discussed `create_and_update` for this.

Reply:
Opps, updated.